### PR TITLE
Add my personal pin location

### PR DESCRIPTION
**Pull request by aperez36**
My name is Alberto "Carlos" Perez. Wanted to pin my location.

### CODE REVIEW

**Comment by aperez36** [comment]
Hey my name is Alberto "**Carlos**". Currently studying **Computer Engineering** at USF and the Tech Director for *SHPE*. Objective is to pin my location. This was all thanks to @githubteacher

File: Aperez36.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Aperez36
+latitude: 25.4321° N
+longitude: 100.9837° W
```
Comment (kokila92) [inline Aperez36.json]: Perfect!

**Comment by brianamarie** [comment]
@aperez36 It looks like you entered your longitude and latitude with some letters at the end. [This link may be helpful to convert your coordinates to the correct format](http://www.earthpoint.us/Convert.aspx). 😄

**Comment by perlba** [comment]
Well done :)

**Pull request merged**